ChronosTankDrive/src/main/java/frc/robot/Robot.java

```diff
@@ -115,8 +115,8 @@ public void robotInit() {
     // result in both sides moving forward. Depending on how your robot's
     // gearbox is constructed, you might have to invert the left side instead.
     // chronosDrive = new DifferentialDrive(leftFrontDrive, rightFrontDrive);
-    leftStick = new Joystick(0);
-    rightStick = new Joystick(2);
+    rightStick = new Joystick(0);
+    leftStick = new Joystick(2);
```

Comment:
FYI - you can reassign which joystick is in which slot in the driver station by following these instructions: https://docs.wpilib.org/en/stable/docs/software/basic-programming/joystick.html#driver-station-joysticks